### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JennieOhyoung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# code_review
\ No newline at end of file
+changes changes changes
```
Comment (drock1) [inline README.md]: add another change

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# code_review
```
Comment (drock1) [inline README.md:1]: suggestion #2

**Review by drock1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (JennieOhyoung) [reply README.md:1]: got this

File: README.md (hunk 1 above)
Comment (JennieOhyoung) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (JennieOhyoung) [inline README.md:1]: comment on commit 1

**Review by gnuoyhoeinnej — APPROVED** [review]

**Review by gnuoyhoeinnej — CHANGES_REQUESTED** [review]
can you make changes with pending request as admin?

**Review by gnuoyhoeinnej — APPROVED** [review]

**Review by gnuoyhoeinnej — APPROVED** [review]

**Review by gnuoyhoeinnej — DISMISSED** [review]
fix this shit

**Review by gnuoyhoeinnej — APPROVED** [review]

**Review by gnuoyhoeinnej — CHANGES_REQUESTED** [review]
found another shitty thing to fix

**Review by gnuoyhoeinnej — APPROVED** [review]

**Review by gnuoyhoeinnej — CHANGES_REQUESTED** [review]
disproves

**Comment by JennieOhyoung** [comment]
test test

**Comment by JennieOhyoung** [comment]
testing comment via email #2

**Pull request closed without merging**